src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp

```diff
@@ -155,7 +155,10 @@
     <br/>
 
     <kra-pd:keyPersons/>
-
+    <div id = "workarea">
+        <kra-pd:proposalDevelopmentQuestionnaireAnswersFiltered bean = "${KualiForm.questionnaireHelper}" property = "questionnaireHelper"/>
```

Comment:
no need to bring all the questions and answers from Questionnaire in this page, just ONE of them, with the id=11018. 
the logic should loop through the questions and answers in the helper and display just this particular one on this page.

Reply:
no need to bring all the questions and answers from Questionnaire in this page, just ONE of them, with the id=11018.
the logic should loop through the questions and answers in the helper and display just this particular one on this page.